torch_frame/nn/encoder/stype_encoder.py

```diff
@@ -174,21 +182,11 @@ def na_forward(self, feat: TensorData) -> TensorData:
         """
         if self.na_strategy is None:
             return feat
-
+        if (isinstance(feat, _MultiTensor) and get_nan_mask(feat.values).any()
+                or isinstance(feat, Tensor) and get_nan_mask(feat).any()):
+            # Only clone the feat if there are NaNs in the data
+            feat = feat.clone()
```

Comment:
minor nit: The `clone` here would also clone offsets, while we would only need to clone the values.

Reply:
Changed the code to only clone the `values` tensor instead.